src/main/groovy/org/pih/warehouse/core/ActivityCode.groovy

```diff
@@ -111,6 +111,9 @@ enum ActivityCode {
     // Inbound sortation
     INBOUND_SORTATION('INBOUND_SORTATION'),
 
+    // Putaway discrepancy
+    PUTAWAY_DISCREPANCY('PUTAWAY_DISCREPANCY'),
```

Comment:
I hadn't thought of using the activity code for this, but couldn't figure out the best way to model this. So I think it's a good idea for now. 

Eventually, we might want to make these locations a little more generic at some point i.e. LOST_AND_FOUND. So we can look for a LOST_AND_FOUND location within the zone where the discrepancy occurs Inbound Sortation Zone (putaway discrepancy) or Storage Zone 1 (picking discrepancy). 

Actually, let's do this. Call the activity code LOST_AND_FOUND. And we can have one Lost & Found location for the whole facility. The actual location will be called Lost & Found and it will be of type Discrepancy Location with activity code (HOLD_STOCK, LOST_AND_FOUND).